backend/src/graphql/resolvers/queries.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const User = require('../../models/user');
+
+module.exports = {
+  users: async () => (users = await User.find()),
+  login: async (_, { email, password }, { res }) => {
+    // lowerCase the email && check if the user exists
+    const user = await User.findOne({ email: email.toLowerCase() });
+    // compare provided password against stored password
+    let rightPassword = false;
+    if (user) {
+      rightPassword = await bcrypt.compare(password, user.password);
+    }
+    if (!user || !rightPassword) {
```

Comment:
Maybe `try` `catch` here, if it errors at any point, the email or password is incorrect.